### ISSUE TITLE
Angel Ring canot be place on baubles slot anymore...

### ISSUE THREAD

**Issue by Fireztonez** [rwtema/extrautilities_old_issues#1152]
Hello,
With my new modpack server on Minecraft 1.10.2 (Forge 1.10.2-12.18.3.2185), I can't place the Angel Ring on any Baubles Slot, I need to keep on my inventory...

Before, on my precedent modpack, with older version of mods, but sill on Minecraft 1.10.2, the Angel Ring Could fit into the Baubles slots.

I have version 1.1.2 from Extra Utilities and Baubles 1.3.6

A bug due to a compatibility problem with a Baubles update ??

**Comment by ghost** [comment]
Yeah. I haven't been able to "wear" the ring either. I expect getting it in the babules' slot is a low priority.

**Comment by Fireztonez** [comment]
Yes i understand your point.

Le mar. 13 déc. 2016 22:50, Charles Evans <notifications@github.com> a
écrit :

Yeah. I haven't been able to "wear" the ring either. I expect getting it in
the babules' slot is a low priority.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/rwtema/extrautilities/issues/1152#issuecomment-266935564>,
or mute the thread

**Comment by ghost** [comment]
Check out the mod [Angel Ring to Bauble](https://minecraft.curseforge.com/projects/angel-ring-to-bauble).
